src/main/java/domain/Number.java

```diff
@@ -20,6 +23,11 @@ private void validate(int number) {
         }
     }
 
+    public static Number generateNumber() {
+        int randomNumber = random.nextInt(LOTTO_NUMBER_MAX) + LOTTO_NUMBER_MIN;
+        return cached.get(randomNumber - 1);
```

Comment:
index는 0 부터 시작하니 아래와 같이 사용할수 있을것 같아요 👍 
```suggestion
        return cached.get(random.nextInt(LOTTO_NUMBER_MAX));
```